pkg/cmd/server/bootstrappolicy/constants.go

```diff
@@ -28,6 +28,8 @@ const (
 	// This should remain in the default role bindings for the NodeAdmin role
 	LegacyMasterKubeletAdminClientUsername = "system:master"
 	MasterKubeletAdminClientUsername       = "system:openshift-node-admin"
+
+	InfraNodeBootstrapServiceAccountName = "system:" + NodeBootstrapRoleName
```

Comment:
k8s defines the name as ``apiconfig.NodeBootstrapperClusterRoleName``. Can we use the name from ``k8s.io/kubernetes/cmd/kubeadm/app/phases/apiconfig``?